### PR TITLE
Consensus API docs

### PR DESCRIPTION
**Pull request by VoidingWarranties**

### CODE REVIEW

File: doc/API.md [hunk 1]
```diff
@@ -184,35 +184,23 @@ struct {
 Consensus
 ---------
 
-Queries:
-
-* /consensus                 [GET]
+| Route                        | HTTP verb |
+| ---------------------------- | --------- |
+| [/consensus](#consensus-get) | GET       |
```
Comment (VoidingWarranties) [inline doc/API.md]: Should have a note pointing to `Consensus.md`. See Gateway for an example.

File: doc/api/Consensus.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+  // Number of blocks preceding the current block.
+  "height": 62248,
+
+  // Hash of the current block.
+  "currentblock": "00000000000008a84884ba827bdc868a17ba9c14011de33ff763bd95779a9cf1",
+
+  // An immediate child block of this block must have a hash less than this
+  // target for it to be valid.
+  "target": [0,0,0,0,0,0,11,48,125,79,116,89,136,74,42,27,5,14,10,31,23,53,226,238,202,219,5,204,38,32,59,165]
```
Comment (DavidVorick) [inline doc/api/Consensus.md]: note to self: we should add 'difficulty' here next to target, since most people find difficulty easier to read.

Comment (DavidVorick) [inline doc/api/Consensus.md]: v1.1 I guess. If you agree let's make an issue on the tracker.

**Comment by DavidVorick** [comment]
looks pretty good to me, except for the note you made for yourself.

**Comment by VoidingWarranties** [comment]
Fixed the one issue and rebased. Should be good to go.

**Pull request merged**